gtsam/hybrid/tests/Switching.h

```diff
@@ -114,11 +114,11 @@ inline std::pair<KeyVector, std::vector<int>> makeBinaryOrdering(
   return {new_order, levels};
 }
 
-/* ***************************************************************************
- */
+/* ****************************************************************************/
 using MotionModel = BetweenFactor<double>;
 
 // Test fixture with switching network.
+/// ϕ(X(0)) .. ϕ(X(k),X(k+1)) .. ϕ(X(k);z_k) .. ϕ(M(0)) .. ϕ(M(k),M(k+1))
```

Comment:
This docstring should not have `M(k+1)` since we need K modes for K+1 continuous variables.
E.g. for K = 2
```
X0 -*- X1 -*- X2
    |      |
    M0     M1
```

Reply:
It says k not K. But I'll fix to K-2.

Reply:
It should only be up to M(k) not M(k+1).